ehrql/measures/measures.py

```diff
@@ -300,6 +306,31 @@ def configure_dummy_data(self, *, population_size):
         """
         self.dummy_data_config.population_size = population_size
 
+    def configure_disclosure_control(self, *, enabled=True):
+        """
+        Configure disclosure control.
+
+        By default, the numerators and denominators of measures generated from real and
+        dummy tables, and by the dummy data generator, are subject to disclosure
+        control. First, values less than or equal to `7` are replaced with `0`
```

Comment:
Took me a couple of goes to parse this.  How about:

> By default, numerators and denominators are subject to disclosure control (unless the user has provided their own dummy data file).

Reply:
I think Peter's rephrasing is clearer. I wonder though if we should just apply SDC to everything, including user-supplied dummy data. It's a better match for what's going to happen in production, and it gives us a simpler story here.

Reply:
I'm going to make the reasoning explicit.

The question: Should the measures framework always respect `measures.disclosure_control_config.enabled`?

* If the answer is "no", then the story is more complex: one route for these kinds of data, another route for this kind of data.
* If the answer is "yes", then the story is simpler. What are the drawbacks?

If a user intentionally supplied an uncontrolled dummy data file, but didn't call `measures.configure_disclosure_control(enabled=False)`, then the measures framework would apply disclosure control to their dummy data. The drawback is that doing so might confuse the user. But as you say, a local run would match a production run: We would want to prompt the user either to change their dummy data or to configure disclosure control appropriately.

That's reasonable: I will update to always respect `measures.disclosure_control_config.enabled`.